docs/tools/index.md

```diff
@@ -7,4 +7,5 @@ maxdepth: 1
 
 sharpness-and-focus-point
 maximum_resolution_achievable
+saturation_indicator
```

Comment:
Is the notebook called `saturation_indicator` missing...?

And `maximum_resolution_achievable` is also missing from the menu: https://remote-unzip.deno.dev/scipp/essimaging/artifacts/3961599931/tools/index.html